ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.1.9f1
+m_EditorVersion: 2018.1.9f2
```

Comment:
Do we want this?

Reply:
Revert this for now. Granted most of us are using f2, but we didn't agree to update it in the project yet. Plus it's not part of this PR

Reply:
do we seriously need a special PR to rev to the latest 2018.1? I can see a wider discussion for moving to .3